sprints/dashboard/utils.py

```diff
@@ -97,6 +101,67 @@ def get_cell_member_names(conn: CustomJira, members: Iterable[str]) -> Dict[str,
     """Returns cell members with their names."""
     return {conn.user(member).displayName: member for member in members}
 
+def get_cell_member_roles(raise_exception: bool = False) -> Dict[str, List[str]]:
+    """Return a dictionary of cell members and their associated roles.
+    Example return: {'John Doe': ['Recruitment Manager', 'Sprint Planning Manager',...],...}.
+
+    Note: We have room for error here if member names are spelled differently, hence we must
+    be careful that the handbook & Jira have consistent naming.
```

Comment:
Maybe something along this would be more verbose?
```
Note: we are using the following sources of data for this function:
  1. Handbook (cell roles, e.g. "Sprint Planning Manager").
  2. Rotations spreadsheet (sprint roles - e.g. "Firefighter", "Discovery Duty")
  3. Jira (matching users with their email addresses).
Therefore we should ensure that users' names don't contain any typos, as this will provide inaccurate results.
```